charts/index.yaml

```diff
@@ -3,19 +3,19 @@ entries:
   azure-service-operator:
   - apiVersion: v2
     appVersion: 0.1.0
-    created: "2020-05-20T10:20:46.240488-06:00"
+    created: "2020-06-01T12:41:48.174129-06:00"
     dependencies:
     - condition: azureUseMI
       name: aad-pod-identity
       repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
```

Comment:
https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts doesnt load for me

Reply:
hmm wonder if it moved...

@cnadolny did the aad pod chart move? seems to be in the `aad-pod-identity` directory under charts

Reply:
This should be okay, as running the command below works:
helm repo add aad-pod-identity https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts

Reply:
Thanks! Looks good. I was able to add the repo as well using that url.